tests/test_logs.py

```diff
@@ -34,6 +34,48 @@ def test_get_log_id(self):
         with self.assertRaises(Exception):
             self.api.get_log_id('1212')
 
+    def test_request_event_csv(self):
+        log_id = self.api.get_log_id('Incident_Management.csv')
+        resp = self.api.request_event_csv(log_id)
+        self.assertEqual(resp.status_code, 200)
+
+        mining_request = {
+            'activityExclusionFilter': [],
+            'includeHeader': True,
+            'includeLogId': False,
+            'logId': log_id,
+            'edgeThreshold': 1,
+            'traceFilterSequence': [],
+            'exportCaseAttributes': True,
+            'onlyColumns': ["Case ID", "Action"],
+            'runConformance': False}
+        resp = self.api.request_event_csv(log_id, mining_request=mining_request)
+        self.assertEqual(resp.status_code, 200)
+
+        resp = self.api.request_event_csv('never-ever-matches-a-log-id')
+        self.assertNotEqual(resp.status_code, 200)
+
+    def test_get_event_log(self):
+        log_id = self.api.get_log_id('Incident_Management.csv')
+        log = self.api.get_event_log(log_id=log_id)
+        self.assertGreater(len(log), 0)
+
+        mining_request = {
+            'activityExclusionFilter': [],
+            'includeHeader': True,
+            'includeLogId': False,
+            'logId': log_id,
+            'edgeThreshold': 1,
+            'traceFilterSequence': [],
+            'exportCaseAttributes': True,
+            'onlyColumns': ["Case ID", "Action"],
+            'runConformance': False}
+        log = self.api.get_event_log(log_id=log_id, mining_request=mining_request)
+        self.assertGreater(len(log), 0)
```

Comment:
It is always nice to give messages to the assertions, that describe the test. I try to always use s.th. like

```python
msg = 'failed to retrieve existing log'
log = self.api.get_event_log(log_id=log_id, mining_request=mining_request)
self.assertGreater(len(log), 0, msg)
```

The message is then shown when the test fails and one can quickly understand what failed. The Testcase (the class) and the method are also mentioned by nam,e, but as soon as there is more than one assertion, the message helps a lot.